tests/unit/cli/test_integration.py

```diff
@@ -83,42 +89,78 @@ def test_integration_install_specific_integration(
     """Tests that the install command behaves as expected when supplied with
     a specific integration"""
     runner = CliRunner()
-    mocker.patch.object(
+
+    mock_install_package = mocker.patch.object(
         sys.modules["zenml.cli.integration"],
         "install_package",
         return_value=None,
     )
+    mocker.patch(
+        "zenml.integrations.registry.IntegrationRegistry.is_installed",
+        return_value=False,
+    )
 
     result = runner.invoke(integration, ["install", "-f", integration_name])
     assert result.exit_code == 0
+    mock_install_package.assert_called()
+
+
+def test_integration_install_multiple_integrations(mocker: MockFixture) -> None:
+    """Tests that the install command behaves as expected when supplied with
+    multiple integration"""
+    runner = CliRunner()
+
+    mock_install_package = mocker.patch.object(
+        sys.modules["zenml.cli.integration"],
+        "install_package",
+        return_value=None,
+    )
+    mocker.patch(
+        "zenml.integrations.registry.IntegrationRegistry.is_installed",
+        return_value=False,
+    )
+
+    result = runner.invoke(integration, ["install", "-f", *INTEGRATIONS])
```

Comment:
Does this work on mark parametrize. As I understand it, mark.parametrize calls the test_function it decorates for each item in the supplied list.

Reply:
As such I would have expected INTEGRATIONS to be a string here, so I would assume this iterates through the string?

Reply:
My bad, I missread something, this function isn't decorated - ignore this